src/qiskit_sphinx_theme/directives.py

```diff
@@ -50,7 +50,7 @@ def run(self) -> list[nodes.paragraph]:
         card_rst = f"""
 .. raw:: html
 
-    <div class="qiskit-card" link={link}>
+    <div class="qiskit-card" onclick="window.location = '{link}';">
```

Comment:
I'm a bit uncertain about how accessible this is. I know it is basically just fixing what was broken from the legacy theme but when it comes to accessibility I think we should leave things better than we find them. E.g. I know it's bad practice to make a div clickable, and its better to use semantic html (e.g. change it to an `<a>` tag) or at least add alt text so that people with screen readers know this div is actually a button